src/__tests__/ReadPerMinute.test.js

```diff
@@ -146,5 +146,29 @@ describe('ReadPerMinute', () => {
 			const instance = new ReadPerMinute()
 			expect(instance.parse(text, lang)).not.toEqual(expected)
 		})
+
+		test.each([
+			[
+				generateTokenizedText(null, 425, 425).str,
+				425,
+				{ time: 1, words: 425, rate: 425 },
+			]
+		])('Parses text and uses the custom rate for calculations', (text, langOrRate, expected) => {
```

Comment:
Naming the argument what it is makes the statement clearer:
```suggestion
		])('Parses text and uses a custom rate', (text, rate, expected) => {
```